pkg/connProfile/get.go

```diff
@@ -0,0 +1,121 @@
+/*
+Copyright 2023 The Bestchains Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package connProfile
+
+import (
+	"context"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"gopkg.in/yaml.v2"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/tools/clientcmd"
+
+	"github.com/bestchains/bc-cli/pkg/common"
+)
+
+func getNestedString(obj map[string]interface{}, fields ...string) string {
```

Comment:
@0xff-dev @dayuy  Do we need this function in other commands ? If yes,we should place this under `common` or `utils`?

Reply:
This function ignores some error handling, other commands may need to do error handling.

Reply:
better to handle error all the time ... even channel profile json retrieved from k8s can be in bad format

Reply:
Errors should be handled to avoid panic.  But `network` and `id` fields are required fields in the channel. So these two fields must not be empty, and the following base64 parsing will return an error message if an error occurs.
So I think we can ignore the error handling here.

Reply:
> better to handle error all the time ... even channel profile json retrieved from k8s can be in bad format

Or we can remove the getNestedString function, which is just one more layer to call. Just call the unstructured related function directly, and leave it to the caller whether to handle the error or not.

Reply:
make sense. since there is no panic at this place,we can just leave this func.